### PR TITLE
Set stack root refactor

### PR DESCRIPTION
**Pull request by guyca**
This PR adds support for setting stack root with multiple children, which is to be expected as stack can be initialised with multiple children.

### CODE REVIEW

**Comment by henrikra** [comment]
Wow nice! Was this the real fix to the PR I started? :D https://github.com/wix/react-native-navigation/pull/4434

File: lib/src/Navigation.ts [hunk 1]
```diff
@@ -146,8 +147,9 @@ export class NavigationRoot {
   /**
    * Sets new root component to stack.
    */
-  public setStackRoot(componentId: string, layout: Layout): Promise<any> {
-    return this.commands.setStackRoot(componentId, layout);
+  public setStackRoot(componentId: string, layout: Layout | Layout[]): Promise<any> {
+    const children: Layout[] = isArray(layout) ? layout : [layout];
```
Comment (henrikra) [inline lib/src/Navigation.ts]: You dont need lodash. You can use  `Array.isArray` more info https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray

**Comment by henrikra** [comment]
Currently testing version `2.2.5-snapshot.82` and looks like this feature is working on iOS and Android!

**Comment by guyca** [comment]
@henrikra [Here's how we use it in the playground app](https://github.com/wix/react-native-navigation/blob/master/playground/src/screens/PushedScreen.js#L170)

**Pull request merged**